tmk_core/protocol/report.h

```diff
@@ -210,8 +216,8 @@ typedef struct {
 #endif
     mouse_xy_report_t x;
     mouse_xy_report_t y;
-    int8_t            v;
-    int8_t            h;
+    mouse_hv_report_t v;
+    mouse_hv_report_t h;
```

Comment:
Has been included in [a separate PR](https://github.com/qmk/qmk_firmware/pull/24422).